Modix/Controllers/ChannelController.cs

```diff
@@ -17,6 +20,12 @@ public ChannelController(DiscordSocketClient client, IAuthorizationService modix
             ChannelService = channelService;
         }
 
+        [HttpGet("types")]
```

Comment:
Nitpick: "ChannelDesignationTypes" would be more consistent for the endpoint name.

Reply:
This should be fine, as the full endpoint ends up being `api/config/channels/types`. `api/config/channels/channeldesignationtypes` seems a bit long.

Reply:
I would ACTUALLY have it be `api/config/channels/designation-types`, but looking at existing endpoints in this controller, we have such things as `api/config/channels/channeldesignations`.

Reply:
Oh, no, those aren't actually named. For example, the route on the controller itself is `[Route("~/api/config/channels")]`, which means that
```csharp
[HttpGet]
public async Task<IActionResult> ChannelDesignations()
```
will just route to any GET request to `/api/config/channels`

Reply:
Riiiiiiiiight. I'd forgotten how utterly unintuitive MVC controller routes are.